src/core/shape_inference/include/pooling_shape_inference_util.hpp

```diff
@@ -151,6 +151,42 @@ void valid_dilated_kernel_with_padding(const TOp* op,
                                        const size_t pad_end,
                                        const size_t axis) {}
 
+template <class TDim>
+void valid_ceil_torch_dimension_size(TDim& dim,
```

Comment:
detail: As this function update the value of the dimension, consider:
```suggestion
void align_ceil_torch_dimension_size(TDim& dim,
```
or
```suggestion
void get_valid_ceil_torch_dimension_size(TDim& dim,
```

Reply:
Done, changed to `align`.